src/features/folder_creation/contest_folder_creation.ts

```diff
@@ -33,8 +33,10 @@ export const createContestDirectory = async (
     catch(err) {
         if(err.code === ErrorCodes.folderExists) {
             vscode.window.showErrorMessage('Contest folder already exists');
-        }else if(err.code ===ErrorCodes.noWritePermission) {
+        }else if(err.code ===ErrorCodes.noAccessPermission) {
```

Comment:
Erroe message here looks good, would be good to have same error message ("Please open a folder in your workspace") in problem folder creations as well